Makefile

```diff
@@ -52,7 +52,7 @@ sql: ## Generate raw sql for the migrations
 	docker-compose run katalogus python -m alembic --config /app/boefjes/boefjes/alembic.ini upgrade $(rev1):$(rev2) --sql
 
 check:
-	pre-commit run --all-files --show-diff-on-failure --color always
```

Comment:
This is actually quite a neat feature from `pre-commit`. Why remove this?

Reply:
Because it does not show the relevant diff only, it shows the complete staged diff which means I spend a lot of time scrolling upward to find the file flake-8 claims to have unused imports, never looking at the diffs...